src/ontology/mondo-edit.obo

```diff
@@ -30213,7 +30213,7 @@ xref: UMLS:C0006840 {source="MONDO:equivalentTo", source="NCIT:C26711", source="
 is_a: MONDO:0002041 {source="DOID:1508/inferred", source="ICD10CM:B37", source="MESH:D002177", source="MONDO:Redundant", source="NCIT:C26711"} ! fungal infectious disease
 is_a: MONDO:0002312 {source="DOID:1508", source="MONDO:Redundant"} ! opportunistic mycosis
 intersection_of: MONDO:0005550 ! infectious disease
-intersection_of: disease_has_infectious_agent NCBITaxon:1535326 ! Candida <clade Candida/Lodderomyces clade>
+intersection_of: disease_has_infectious_agent NCBITaxon:1535326
```

Comment:
Change suggests taxon is missing in import

Reply:
done!